cmd/kubectl_token.go

```diff
@@ -489,55 +497,54 @@ func samlAuth(input *LoginInput, client *http.Client) (managementClient.Token, e
 			}
 			token.Token = string(decryptedBytes)
 
-			// delete token
-			req, err = http.NewRequest(http.MethodDelete, tokenURL, bytes.NewBuffer(nil))
-			if err != nil {
-				return token, err
+			// Delete the auth token.
+			resp, _, err := doRequest(client, deleteReq)
+			if err == nil && resp.StatusCode != http.StatusOK && resp.StatusCode != http.StatusNotFound {
+				err = fmt.Errorf("unexpected http status code %d", resp.StatusCode)
 			}
-			req.Header.Set("content-type", "application/json")
-			req.Header.Set("accept", "application/json")
-
-			client.Timeout = 150 * time.Second
```

Comment:
Same here. There is nothing specific or unusual about this request that might require a custom timeout.